platform/lib/middleware/redirects.test.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const request = require('supertest');
+
+// The languages used here have to be in the config.js
+// We do not mock the config, since it is not likely to change
+const VALID_LANGUAGE_SHORT = 'en';
+const VALID_LANGUAGE_LONG = 'pt_br';
+
+const UNIT_TEST_REDIRECTS = '/shortcut: /some/deep/link/\n' +
```

Comment:
nit: JS template strings work better here

```
const UNIT_TEST_REDIRECTS = `
/shortcut: /some/deep/link/
/${VALID_LANGUAGE_SHORT}/old/link: /new/link/target
`
```

Reply:
Good idea!
I changed that.